plugin-name/composer.json

```diff
@@ -1,12 +1,13 @@
 {
    "name": "wpbp/wordpress-plugin-boilerplate-powered",
    "description": "Wordpress Plugin Boilerplate with steroids!",
+   "type": "wordpress-plugin",
    "license": "GPL-2.0-or-later",
    "authors": [
       {
          "name": "author_name",
-         "email": "author@email.it",
-         "homepage": "http://author.url"
+         "email": "author@example.com",
+         "homepage": "https://author-url.test"
```

Comment:
This change will require to update the generator https://github.com/WPBP/generator/blob/ed2fed4d13acaf2c94eb740cb982fa944c5cdd38/generator/composer.php#L224
Also for the email, can you do a pr also for that?

Reply:
Done.
https://github.com/WPBP/generator/pull/46